microsoft-edge/devtools-guide-chromium/sources/index.md

```diff
@@ -22,62 +22,231 @@ keywords: microsoft edge, web development, f12 tools, devtools
+
+A Workspace works well when the JavaScript code that's returned by the server is the same as your local JavaScript source code that you maintain, rather than performing minification, transformation, or other compiling of JavaScript code.
+
+When using a Workspace, changes to your front-end code are preserved across page refreshes.
+
+For more information, navigate to the tutorial [Edit files with Workspaces][DevtoolsGuideChromiumWorkspacesIndex].
+
+#### Sources tool without a Workspace
+
+In contrast to using a Workspace, when you use the Sources tool in the default way, you edit a transient copy of the front-end code that's returned by the web server.  This non-Workspace approach supports fast iterations when you are prototyping JavaScript code and don't want to go through the build process every time you want to view the rendered result of a code edit.
+
+By default, when you edit a file in the **Sources** tool, those changes are lost when you refresh the page.  In the default approach, the **Sources** tool works with a copy of the client-side resources that are returned by the web server.  When you modify these client-side files that are returned by the server, the changes don't persist, because you didn't change the source files.
+
+When you find a fix that works for the client-side JavaScript that is returned by the server, you then need to apply that fix, or an equivalent of that fix, in your actual source code and then re-deploy to the server.
+
+#### Mapping minified code to your source code, to work with readable source code
+
+<!-- From a user point of view, is Workspace related to mapping minified code to source code? -->
+
+If your client-side source code is transformed, so that the server returns different, transformed, hard-to-read (minified) front-end code to the browser, in some cases, you can set up mapping from your source code to your compiled code, so that when you view, debug, and edit the transformed client-side JavaScript code that's returned from the server, the code looks familiar and readable.
+
+Source Maps from preprocessors cause DevTools to load your original JavaScript source files in addition to your minified, transformed JavaScript files that are returned by the server.  You then use your original source files to set breakpoints and step through code.  Meanwhile, Microsoft Edge is actually running your minified code. 
+
+The mapping approach keeps your client-side code readable and debuggable even after you combine, minify, or compile it.
+For more information, navigate to [Map preprocessed code to source code][DevToolsJavaScriptSourceMaps].
+
+#### Complex transformations from source code to compiled front-end code
+
+<!-- todo: Does this section have to do with Workspace?  if not, maybe move this section elsewhere, to preserve the series of 5 sections for the 5 tabs of the Explorer pane. -->
```

Comment:
@zoherghadyali - Does this section (Complex transformations from source code to compiled front-end code) have to do with Workspace? if not, maybe move this section elsewhere